src/pipe.toit

```diff
@@ -29,8 +29,19 @@ UNKNOWN-DIRECTION_ ::= 0
 PARENT-TO-CHILD_ ::= 1
 CHILD-TO-PARENT_ ::= 2
 
+interface StreamOrPipe implements old-reader.Reader:
```

Comment:
To me, a pipe is a stream. I think the public names here are kinda sucky:

Stream, StreamOrPipe, OpenPipe.

It feels almost like these would be better in particular if you can use Stream pretty much everywhere.

OpenFile, Stream, OpenPipe.

Maybe you should add `is-a-terminal` to StreamOrPipe.

Reply:
I don't see how to make this change without breaking users.
I think this has to wait for a v2 (which this package needs anyway).

I added `is-a-terminal`.